### PR TITLE
fix: Don't crash installation when saving last APK is disabled

### PR DESCRIPTION
**Pull request by BenjaminHalko**
closes: #2125

### CODE REVIEW

File: lib/ui/widgets/settingsView/settings_last_patched_app.dart
Comment (Axelen123) [inline lib/ui/widgets/settingsView/settings_last_patched_app.dart:1]: Why is this here?

**Review by Axelen123 — CHANGES_REQUESTED** [review]

File: lib/ui/widgets/settingsView/settings_last_patched_app.dart
Comment (BenjaminHalko) [reply lib/ui/widgets/settingsView/settings_last_patched_app.dart:1]: The setting didn't have haptics applied, so I figured I'd fix it since I was fix a last Patched App issue anyways. Though maybe it is better to place it in its own PR?

Comment (BenjaminHalko) [reply lib/ui/widgets/settingsView/settings_last_patched_app.dart:1]: I've already created a separate PR for another last apk issue, so maybe I'll do the same for this.

**Review by Axelen123 — APPROVED** [review]
I don't know if the pubspec.lock changes belong here or not, but I don't have anything else to comment on.
